MyApplication/app/src/main/java/com/example/myapplication/NewActivity.kt

```diff
@@ -1,23 +1,27 @@
 package com.example.myapplication
 
+import android.R.attr.button
 import android.app.Activity
 import android.content.Intent
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import android.widget.Button
 import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+
 
 class NewActivity : AppCompatActivity() {
     lateinit var backButton: Button
+    lateinit var nextButton: Button
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_new)
         title = "MyApp"
         val message = intent.getStringExtra("message_key")
         val messageTextView: TextView = findViewById(R.id.textView)
-        messageTextView.text = "$message, glad to see you here!"
+        messageTextView.text = "Dear $message, welcome to the Authentic Social Network! The purpose of this application is to serve as a way to connect individuals with other people in order to share information, ideas, messages, creativity and interests. Authentic Social Network has no likes, followers and influencers, it's a platform for network & inspiration where mental health is a priority!"
```

Comment:
Move this string to string resource xml file

Reply:
moved